types/dns/rfc/fqdn.pp

```diff
@@ -0,0 +1 @@
+type Stdlib::DNS::Rfc::Fqdn = Stdlib::Fqdn
```

Comment:
If I understand your point, the actual Pattern from `Stdlib::Fqdn` should be moved here, and `Stdlib::Fqdn` should equal to `Stdlib::DNS::Rfc::Fqdn`.

Reply:
yes i think that makes senses, updated